lib/config/presets/index.ts

```diff
@@ -336,8 +338,16 @@ export async function resolveConfigPresets(
           fetchedPreset,
           baseConfig ?? inputConfig,
           ignorePresets,
-          existingPresets.concat([preset])
+          existingPresets.concat([preset]),
+          false
         );
+        if (preset.includes('>')) {
```

Comment:
I'm not sure this is the safest way and robust enough to distinguish between internal vs external presets